#160452438 User should be able to create an article

Alameen688 opened the pull request:
#### What does this PR do?
Implements create user form design and allow a user to create an article
#### Description of Task to be completed?
- Include current design for the create article page
- Add publish settings to include tags
- Allow a user to create an article
#### How should this be manually tested?
- Checkout to this branch `git checkout ft-160452438-create-article`
- Navigate to the axio.js file in the services directory, set the value of the token constant to a valid user token
- RUN `npm start`
- Navigate to [http://localhost:3000/article/add](http://localhost:3000/article/add)
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-11-03 at 10 23 02 am" src="https://user-images.githubusercontent.com/9039613/47950557-2ec96e80-df54-11e8-8cc5-47cbba36a785.png">


#### Questions:

Alameen688 committed:
feat(create-article): User should be able to create an article
- adds tests
- contains pending test
- includes current design for the create article page
- allow user to create article

waleCloud commented on src/components/article/createArticle/createArticle.js:
```diff
@@ -0,0 +1,101 @@
+import './createArticle.scss';
+
+export class CreateArticle extends Component {
+  state = {
```
Kindly take state management to redux.

Alameen688 replied:
Alright, I will do that. Thank you.

waleCloud reviewed (commented):
Increase test coverage, endeavor to cover all necessary files created/modified. before raising a final PR.
well done so far! 👍

Alameen688 committed:
feat(landing-page): add responsive header and articles section (#15)

- add shallow render tests
- add responsive header
- add responsive ArticleGridView with article cards
- fetch and render articles from backend into ArticleGridView

[Partially Delivers #161589741]

uchemukolo committed:
feature(UI): Create login page (#14)

- Design login modal
- Added validations on Login form
- Consumed backend api for login an existing user

[Delivers #160452435]

macphilips committed:
feature(UI): Create sign up page (#16)

- Added unit and integrated for SignUp Component
- Added e2e test for the sign up page
- Added validations on sign up form
- Consumed backend api for creating users
- Added Component for displaying Toast or Alert notifications
- Added Social Sign up button

[Delivers #160452433]

macphilips committed:
feature(UI): Create sign up page (#19)

- Added unit and integrated for SignUp Component
- Added e2e test for the sign up page
- Added validations on sign up form
- Consumed backend api for creating users
- Added Component for displaying Toast or Alert notifications
- Added Social Sign up button

[Delivers #160452433]

Alameen688 commented:
This PR no longer needs to be raised separately against ft-160452438-create-update-article from ft-160452438-create-article. The branch ft-160452438-create-update-article now contains all required file contents to be directly raised against staging.
